Userland/Services/LookupServer/LookupServer.cpp

```diff
@@ -107,7 +107,8 @@ void LookupServer::load_etc_hosts()
         if (original_line.is_empty())
             break;
         auto trimmed_line = original_line.view().trim_whitespace();
-        auto fields = trimmed_line.split_view('\t', false);
+        auto replaced_line = trimmed_line.replace(" ", "\t", true);
```

Comment:
Does this also work when using 4 spaces to separate IP address and hostname?

Reply:
Short answer is yes. Long answer is that it did before the patch too. Tested with several spaces, several tabs and a combination of both.